README.md

```diff
@@ -25,6 +25,7 @@ Name     | Description | Enabled by default
 [service](docs/collector.service.md) | Service state metrics | &#10003;
 [system](docs/collector.system.md) | System calls | &#10003;
 [tcp](docs/collector.tcp.md) | TCP connections |
+[aspnet_aspnetapplications](docs/collector.aspnet_aspnetapplications.md) | ASP.NET Applications |
```

Comment:
I think the name could be shortened to at least `aspnet_application`, possibly even just `aspnet`?